client/src/pages/Achieves/utils.ts

```diff
@@ -0,0 +1,131 @@
+import { GameCategories } from "../../interfaces/Categories";
+import { GameAttempt, User } from "../../interfaces/User";
+import { AchievesId } from "./types";
+
+export const checkIsAchieveOpen = (user: User | null, achieveId: AchievesId) => {
+  if (user) {
+    const stats = user.statistics;
+
+    switch (achieveId) {
+      case AchievesId.FRIENDS:
+        return checkFriendAchieve(user);
+
+      case AchievesId.DISCOVERER:
+        return checkDiscovererAchieve(stats);
+
+      case AchievesId.EXPLORER:
+        return checkExplorerAchieve(stats);
+
+      case AchievesId.SHELDON:
+        return countAttemtsOnCategory(stats, GameCategories.MEMORY) >= 10;
```

Comment:
Looks like 10, 30 and 50 are the magic numbers. Can we use enum instead?

Reply:
Used constants instead